src/Uno.UI.Tests/Windows_UI_XAML_Controls/GridTests/GivenGrid_And_Margin.cs

```diff
@@ -24,150 +17,226 @@ public void When_One_Child_With_Margin_5()
 		{
 			var SUT = new Grid() { Name = "test" };
 
-			var c1 = SUT.AddChild(
-				new View
+			var c1 = new View
 				{
 					Name = "Child01",
-					RequestedDesiredSize = new Windows.Foundation.Size(10, 10),
 					Margin = new Thickness(5)
 				}
-				.GridPosition(0, 0)
-			);
+				.GridPosition(0, 0);
+
+			SUT.AddChild(c1);
+
+			SUT.Measure(new Size(20, 20));
 
-			SUT.Measure(new Windows.Foundation.Size(20, 20));
-			var measuredSize = SUT.DesiredSize;
-			Assert.AreEqual(new Windows.Foundation.Size(20, 20), measuredSize);
-			Assert.AreEqual(new Windows.Foundation.Size(10, 10), c1.RequestedDesiredSize);
+			SUT.DesiredSize.Should().Be(new Size(10, 10));
+			SUT.UnclippedDesiredSize.Should().Be(new Size(10, 10));
+			c1.DesiredSize.Should().Be(new Size(10, 10));
+			c1.UnclippedDesiredSize.Should().Be(new Size(0, 0)); // UnclippedDesiredSize excludes margins
 
-			SUT.Arrange(new Windows.Foundation.Rect(0, 0,20, 20));
+			SUT.Arrange(new Rect(0, 0,20, 20));
 
-			Assert.AreEqual(new Windows.Foundation.Rect(5, 5, 10, 10), c1.Arranged);
+			SUT.Arranged.Should().Be((Rect)"0,0,20,20");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal '0,0,20,20' 4 times.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826507)